src/CommandingOfficers/Commander.java

```diff
@@ -49,7 +49,7 @@ public class Commander implements GameEventListener, Serializable, UnitModifierW
   public static final int CHARGERATIO_FUNDS = 9000; // quantity of funds damage to equal 1 unit of power charge
   public static final int CHARGERATIO_HP = 100; // Funds value of 1 HP damage dealt, for the purpose of power charge
   public int incomeAdjustment = 0; // Commander subclasses can increase/decrease income if needed.
-  private double myAbilityPower = 0;
+  private int myAbilityPower = 0;
```

Comment:
Why ints now? Moving us over to use proper stars? I did notice that term in the Ave change (though I haven't looked at most of this yet).

Reply:
Funds, actually. My thinking is that "real charge" will be in funds (since funds are the smallest fraction of a "star"), and we can just define conversion rates to and from that.
I think I ran into another case of rounding error. I hate rounding error.